### ISSUE TITLE
nginx OR chef_nginx

### ISSUE THREAD

**Issue by athreyavc1** [laradji/zabbix#223]
Hi, 
I was testing the cookbook "chef_nginx" version 5.0.7.  I also have the zabbix cookbook installed. 
Zabbix cookbook has the "nginx" cookbook as dependency. I have experienced failure when zabbix and chef_nginx both are used, due to two versions nginx cookbooks being present. Is it wise to modify the metadata.rb of zabbix cookbook to "depends 'chef_nginx'" ?

**Comment by chewi** [comment]
See #221.

**Comment by chewi** [comment]
True, you should not mix them, but things have since been merged back into nginx.
